core/schemas/entity.py

```diff
@@ -179,13 +185,25 @@ class CourseOfAction(Entity):
     EntityType.vulnerability: Vulnerability,
 }
 
+TYPE_VALIDATOR_MAP = {}
 
 REGEXES_ENTITIES = {
-    EntityType.vulnerability: re.compile(
+    EntityType.vulnerability: ('name', re.compile(
         r"(?P<pre>\W?)(?P<search>CVE-\d{4}-\d{4,7})(?P<post>\W?)"
-    )
+    ))
 }
 
+def validate_entity(ent: Entity) -> bool:
+    if ent.type in TYPE_VALIDATOR_MAP:
+        return TYPE_VALIDATOR_MAP[ent.type](ent) is True
+    elif ent.type in dict(REGEXES_ENTITIES):
+        for field, regex in REGEXES_ENTITIES[ent.type]:
+            if regex.match(getattr(ent, field)):
+                return True
+        return False
+    else:
+        return False
```

Comment:
```suggestion
    return False
```

Reply:
Fixed in d1721b3cbae7adf9097fc56df7f6e0a12ee410a7